src/components/endpoints/Assets.vue

```diff
@@ -93,6 +97,26 @@ const create = async () => {
           deployedAsset.LSP8IdentifiableDigitalAsset
         )
         tokenAddress.value = deployedAsset.LSP8IdentifiableDigitalAsset.address
+        const deployedContract = contract(
```

Comment:
Shouldn't this be something that lsp factory does under the hood when creating new LSP8? cc @Hugoo

Reply:
Well I guess, but as LSP factory has not been touched since a while, it's not up to date on the keys to set

Reply:
Alright, regarding PR lgtm then.

@CallumGrindle @Hugoo maybe we could address this issue at some point.

Reply:
@samuel-videau @dzbo Please create an issue for things like this so we have the details and keep track :)